clm_data/README.md

```diff
@@ -131,6 +142,12 @@ value of 0.0 if C4 is dominant.
   - Finds the rooting beta parameter for the dominant pft, and then calculates and maps the `rooting_depth` parameter
   - Outputs the mapped parameters to `vegetation_properties_map.nc`.
 
+### 3. `soil_variables.py`
+- **Description**: Script for mapping soil properties based on the soil color.
+- **Functionality**:
+  - Finds the soil color for each grid cell using `surfdata_0.9x1.25_16pfts__CMIP6_simyr2000_c170616.nc`.
+  - Maps soil color at each grid cell to dry PAR albedo, dry NIR albedo, wet PAR alebdo, and wet NIR albedo
+
 ## References
 For additional context on the development and capabilities of the Community Land Model, refer to the following publication:
 - Lawrence, D. M., Fisher, R. A., Koven, C. D., Oleson, K. W., Swenson, S. C., Bonan, G., et al. (2019). The Community Land Model Version 5: Description of New Features, Benchmarking, and Impact of Forcing Uncertainty. *Journal of Advances in Modeling Earth Systems*, 11(12), 4245–4287. [DOI:10.1029/2018MS001583](https://doi.org/10.1029/2018MS001583)
```

Comment:
You may add this extra paper for reference: 

Braghiere, R. K., Wang, Y., Gagné-Landmann, A., Brodrick, P. G., Bloom, A. A., Norton, A. J., Ma, S., Levine, P., Longo, M., Deck, K., Gentine, P., Worden, J. R., Frankenberg, C., & Schneider, T. (2023). The Importance of Hyperspectral Soil Albedo Information for Improving Earth System Model Projections. AGU Advances, 4(4), e2023AV000910. https://doi.org/10.1029/2023AV000910

Reply:
This is now added